sdk/python/kfp/components/_structures.py

```diff
@@ -479,6 +482,7 @@ def __init__(self,
         k8s_pod_options: Optional[v1.PodArgoSubset] = None,
     ):
         super().__init__(locals())
+        #TODO: If component_ref is resolved to component spec, then check that the arguments correspond to the inputs
```

Comment:
nit: do you need a name or a ticket number in the TODO? 
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#312-todo-comments